public/views/team.ejs

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <%- include('../components/meta.ejs') %>
+    </head>
+    <body class="vh-100 d-flex flex-column">
+        <header>
+            <%- include('../components/navbar.ejs', {loggedIn, firstName}) %>
+        </header>
+        <main class="d-flex flex-grow-1 bg-body-secondary overflow-hidden">
+            <%- include('../components/sidebar.ejs') %>
+            <div class="container mt-5 content overflow-y-auto overflow-x-hidden">
+              <%- include('../components/actionbar.ejs', {firstName, context: 'user', page: 'team'}) %>
+
+                <div class="d-flex justify-content-between">
+                    <h3>Search for Team Members</h3>
+                    <p>Search for users by their First Name or Email</p>
+                    <form action="#">
+                        <label for="searchUser">Enter Search Here:</label>
+                        <input type="search" id="searchUser" name="searchUser">
+                        <button type="submit" class="btn btn-primary">Search</button>
+                    </form>
+                </div>
+
```

Comment:
The team page might be better served similar to the projects page, which is basically a grid of cards. Each card would represent a team that the user already belongs to, clicking on it would navigate directly to that team's page.

Reply:
The team page that I am starting with is one to allow a user in a project search for team members and then add them to the team/project.